$OpenBSD: patch-xpra_scripts_server_py,v 1.1.1.1 2010/09/06 08:21:09 jasper Exp $

Remove Linux-ism. /proc/self/fd is usually linked to /dev/fd/ anyway.

--- xpra/scripts/server.py.orig	Sun Sep  5 19:27:29 2010
+++ xpra/scripts/server.py	Sun Sep  5 19:46:36 2010
@@ -194,8 +194,8 @@ def run_server(parser, opts, mode, xpra_file, extra_ar
         os.setsid()
         if os.fork():
             os._exit(0)
-        if os.path.exists("/proc/self/fd"):
-            for fd_str in os.listdir("/proc/self/fd"):
+        if os.path.exists("/dev/fd"):
+            for fd_str in os.listdir("/dev/fd"):
                 try:
                     fd = int(fd_str)
                     if fd != logfd:
